build.sbt

```diff
@@ -167,7 +166,7 @@ lazy val supportRojomaV3 = support("rojoma-v3")
   .settings(libraryDependencies += "com.rojoma" %% "rojoma-json-v3" % "3.8.0")
 
 lazy val supportSpray = support("spray")
-  .settings(crossScalaVersions := allCrossVersions)
+  .settings(crossScalaVersions := stableCrossVersions)
```

Comment:
https://github.com/spray/spray-json/issues/273

spray-json for Scala 2.13.0-M5 released

Reply:
Fixed it in c6f7e74.